packages/video_player_videohole/tizen/src/video_player.cc

```diff
@@ -265,6 +265,217 @@ void VideoPlayer::SetPlaybackSpeed(double speed) {
   }
 }
 
+flutter::EncodableList VideoPlayer::getTotalTrackInfo() {
+  player_state_e state = PLAYER_STATE_NONE;
+  int ret = player_get_state(player_, &state);
+  if (ret != PLAYER_ERROR_NONE) {
+    LOG_ERROR("[VideoPlayer] player_get_state failed: %s",
+              get_error_message(ret));
+  }
+  if (state == PLAYER_STATE_NONE || state == PLAYER_STATE_IDLE) {
+    LOG_ERROR("[VideoPlayer] Player not ready.");
+    return {};
+  }
+
+  int video_count = 0;
+  int audio_count = 0;
+  int subtitle_count = 0;
+  int stream_counter = 0;
+  int total_count = 0;
+
+  player_video_track_info_v2 *video_track_info = NULL;
+  player_audio_track_info_v2 *audio_track_info = NULL;
+  player_subtitle_track_info_v2 *sub_track_info = NULL;
+  StreamInformation *streamInfo = NULL;
+
+  void *player_lib_handle = dlopen("libcapi-media-player.so.0", RTLD_LAZY);
+  if (player_lib_handle) {
+    FuncPlayerGetTrackCountV2 player_get_track_count_v2 =
+        reinterpret_cast<FuncPlayerGetTrackCountV2>(
+            dlsym(player_lib_handle, "player_get_track_count_v2"));
+    FuncPlayerGetVideoTrackInfoV2 player_get_video_track_info_v2 =
+        reinterpret_cast<FuncPlayerGetVideoTrackInfoV2>(
+            dlsym(player_lib_handle, "player_get_video_track_info_v2"));
+    FuncPlayerGetAudioTrackInfoV2 player_get_audio_track_info_v2 =
+        reinterpret_cast<FuncPlayerGetAudioTrackInfoV2>(
+            dlsym(player_lib_handle, "player_get_audio_track_info_v2"));
+    FuncPlayerGetSubtitleTrackInfoV2 player_get_subtitle_track_info_v2 =
+        reinterpret_cast<FuncPlayerGetSubtitleTrackInfoV2>(
+            dlsym(player_lib_handle, "player_get_subtitle_track_info_v2"));
+
+    ret = player_get_track_count_v2(player_, PLAYER_STREAM_TYPE_VIDEO,
+                                    &video_count);
```

Comment:
Suggestion to :
```cpp
  int video_count = 0;
  ret = player_get_track_count_v2(player_, PLAYER_STREAM_TYPE_VIDEO,
                                    &video_count);
```
Define the variables when used.

Reply:
fixed!
[Mino code cleanups](https://github.com/flutter-tizen/plugins/pull/607/commits/e918614978b9126d8af37cacaa8fd9b3421288d9)